src/Setup/WixBA/RootView.xaml

```diff
@@ -1,125 +1,487 @@
+                           Margin="0,0,10,10"
+                           HorizontalAlignment="Stretch"
+                           DockPanel.Dock="Bottom">
+                    <Grid x:Name="ActionGrid" HorizontalAlignment="Stretch">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="auto" />
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="5" />
+                            <ColumnDefinition Width="auto" />
+                            <ColumnDefinition Width="*" MinWidth="30" />
+                            <ColumnDefinition Width="auto" />
+                            <ColumnDefinition Width="5" />
+                        </Grid.ColumnDefinitions>
+
+                        <StackPanel x:Name="LeftMarqueeDcIsland"
+                                    Grid.Column="1"
+                                    HorizontalAlignment="Left"
+                                    VerticalAlignment="Center"
+                                    DataContext="{Binding UpdateViewModel}">
+                            <StackPanel x:Name="CheckingUpdatesProgressStjPnl"
+                                        MinWidth="150"
+                                        HorizontalAlignment="Stretch"
+                                        VerticalAlignment="Center">
+                                <ProgressBar x:Name="CheckingUpdatesProgress"
+                                             MinHeight="10"
+                                             Margin="20,0"
+                                             VerticalAlignment="Center"
+                                             HorizontalContentAlignment="Stretch"
+                                             IsIndeterminate="{Binding CheckingEnabled}"
+                                             Style="{DynamicResource UpdateMarqueeStyle}"
+                                             Visibility="{Binding CheckingEnabled,
+                                                                  Converter={StaticResource BooleanToVisibilityConverter}}" />
+                            </StackPanel>
+                        </StackPanel>
+
+                        <StackPanel x:Name="InitiateActionArea"
+                                    Grid.Column="3"
+                                    HorizontalAlignment="Right"
+                                    Orientation="Horizontal">
+                            <!--  Displayed from left to right, when ordered from top to bottom.  Most preferred is at the top.  -->
+                            <StackPanel x:Name="UpdateBtnDcIsland"
+                                        DataContext="{Binding UpdateViewModel}"
+                                        Style="{DynamicResource ActionBtnStkPnlStyle}">
+                                <Button AutomationProperties.HelpText="An update is availalbe. Click to get the update"
```

Comment:
Thanks!!

On Sun, Jul 31, 2016 at 11:56 PM, Sean Hall <notifications@github.com>
wrote:

> In src/Setup/WixBA/RootView.xaml
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r72924125>:
>
> > +                                             IsIndeterminate="{Binding CheckingEnabled}"
> > +                                             Style="{DynamicResource UpdateMarqueeStyle}"
> > +                                             Visibility="{Binding CheckingEnabled,
> > +                                                                  Converter={StaticResource BooleanToVisibilityConverter}}" />
> > +                            </StackPanel>
> > +                        </StackPanel>
> > +
> > +                        <StackPanel x:Name="InitiateActionArea"
> > +                                    Grid.Column="3"
> > +                                    HorizontalAlignment="Right"
> > +                                    Orientation="Horizontal">
> > +                            <!--  Displayed from left to right, when ordered from top to bottom.  Most preferred is at the top.  -->
> > +                            <StackPanel x:Name="UpdateBtnDcIsland"
> > +                                        DataContext="{Binding UpdateViewModel}"
> > +                                        Style="{DynamicResource ActionBtnStkPnlStyle}">
> > +                                <Button AutomationProperties.HelpText="An update is availalbe. Click to get the update"
>
> Need to fix availalbe to available.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/28721ab8e4985a5641b3efd7975a67cecd8f8d4a#r72924125>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLX6vSQWAjCOv66fKZd0LxWnXMFI6ks5qbXxigaJpZM4JVm8P>
> .
>